web_widget_product_label_section_and_note/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Web widget product label section and note",
+    "version": "16.0.1.0.0",
+    "summary": "unify the product and name into a single column",
+    "author": "Tecnativa, Odoo Community Association (OCA), Odoo S.A.",
+    "website": "https://github.com/OCA/web",
+    "depends": [
+        "web",
+        "account",
```

Comment:
This shouldn't depend on `account`. This have to be just the widget.

Reply:
This dependency is necessary because the base component is created in the `account` module, please see:
https://github.com/odoo/odoo/tree/16.0/addons/account/static/src/components/section_and_note_fields_backend

Reply:
OK, understood.